language/de.json

```diff
@@ -0,0 +1,372 @@
+	"Infographic Drawing": "Infografik-Zeichnung",
+	"Ink Dripping Drawing": "Tintentropfen-Zeichnung",
+	"Japanese Ink Drawing": "Japanische Tintenzeichnung",
+	"Knolling Photography": "Knolling-Fotografie",
+	"Light Cheery Atmosphere": "Helle, fröhliche Atmosphäre",
+	"Logo Design": "Logodesign",
+	"Luxurious Elegance": "Luxuriöse Eleganz",
+	"Macro Photography": "Makrofotografie",
+	"Mandola Art": "Mandola-Kunst",
+	"Marker Drawing": "Markerzeichnung",
+	"Medievalism": "Mittelalterstil",
+	"Minimalism": "Minimalismus",
+	"Neo Baroque": "Neo-Barock",
+	"Neo Byzantine": "Neo-Byzantinisch",
+	"Neo Futurism": "Neo-Futurismus",
+	"Neo Impressionism": "Neo-Impressionismus",
+	"Neo Rococo": "Neo-Rokoko",
+	"Neoclassicism": "Neoklassizismus",
+	"Op Art": "Op-Art",
+	"Ornate And Intricate": "Verziert und Komplex",
+	"Pencil Sketch Drawing": "Bleistiftskizze",
+	"Pop Art 2": "Pop Art 2",
+	"Rococo": "Rokoko",
+	"Silhouette Art": "Silhouettenkunst",
+	"Simple Vector Art": "Einfache Vektorgrafik",
+	"Sketchup": "Sketchup",
+	"Steampunk 2": "Steampunk 2",
+	"Surrealism": "Surrealismus",
+	"Suprematism": "Suprematismus",
+	"Terragen": "Terragen",
+	"Tranquil Relaxing Atmosphere": "Ruhige Entspannende Atmosphäre",
+	"Sticker Designs": "Aufkleber-Designs",
+	"Vibrant Rim Light": "Lebendiges Randlicht",
+	"Volumetric Lighting": "Volumetrische Beleuchtung",
+	"Watercolor 2": "Aquarell 2",
+	"Whimsical And Playful": "Launisch und Verspielt",
+	"Model": "Modell",
+	"Base Model (SDXL only)": "Basis-Modell (nur SDXL)",
+	"sd_xl_base_1.0_0.9vae.safetensors": "sd_xl_base_1.0_0.9vae.safetensors",
+	"bluePencilXL_v009.safetensors": "bluePencilXL_v009.safetensors",
+	"bluePencilXL_v050.safetensors": "bluePencilXL_v050.safetensors",
+	"DreamShaper_8_pruned.safetensors": "DreamShaper_8_pruned.safetensors",
+	"realisticStockPhoto_v10.safetensors": "realisticStockPhoto_v10.safetensors",
+	"realisticVisionV51_v51VAE.safetensors": "realisticVisionV51_v51VAE.safetensors",
+	"sd_xl_refiner_1.0_0.9vae.safetensors": "sd_xl_refiner_1.0_0.9vae.safetensors",
+	"Refiner (SDXL or SD 1.5)": "Veredler (SDXL oder SD 1.5)",
+	"None": "Keine",
+	"LoRAs": "LoRAs",
+	"SDXL LoRA 1": "SDXL LoRA 1",
+	"sd_xl_offset_example-lora_1.0.safetensors": "sd_xl_offset_example-lora_1.0.safetensors",
+	"3d_render_style_xl.safetensors": "3d_render_style_xl.safetensors",
+	"Bloodstained-XL-V1.safetensors": "Bloodstained-XL-V1.safetensors",
+	"SDXL_FILM_PHOTOGRAPHY_STYLE_BetaV0.4.safetensors": "SDXL_FILM_PHOTOGRAPHY_STYLE_BetaV0.4.safetensors",
+	"SDXL LoRA 2": "SDXL LoRA 2",
+	"SDXL LoRA 3": "SDXL LoRA 3",
+	"SDXL LoRA 4": "SDXL LoRA 4",
+	"SDXL LoRA 5": "SDXL LoRA 5",
+	"Refresh": "Aktualisieren",
+	"\ud83d\udd04 Refresh All Files": "\ud83d\udd04 Alle Dateien aktualisieren",
+	"Sampling Sharpness": "Proben-Schärfe",
+	"Higher value means image and texture are sharper.": "Ein höherer Wert bedeutet, dass Bild und Textur schärfer sind.",
+	"Guidance Scale": "Führungsmaßstab",
```

Comment:
Edit Pls:
"Guidance Scale": "Führungsmaßstab",
TO:
"Guidance Scale": "Orientierungsskala",